### PR TITLE
coverage added

### LINKED ISSUE #16: Coverage report in every pull request and coverage badge in README.md
https://github.com/marketplace/actions/python-coverage-comment

### PR DESCRIPTION
**Pull request by pychik**
fixes #16

### CODE REVIEW

**Comment by pychik** [comment]
tested several coverage packages. The only one that had started in OK mode was CodeCov. 
Need to merge for more statik info and badge beacuse it is connected to master. created in c5c169f658f285ec3c23712555438adca8c11abd

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# [tool:pytest]
```
Comment (vania-pooh) [inline setup.cfg]: Do we need this file?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -26,3 +26,9 @@ jobs:
       run: |
         chmod +x ci/test.sh
```
Comment (vania-pooh) [inline .github/workflows/test.yml]: Useless `chmod` command.

**Review by vania-pooh — CHANGES_REQUESTED** [review]

File: setup.cfg (hunk 1 above)
Comment (pychik) [reply setup.cfg]: fixed in 6cec3ca1417d268a544349d1f4a3c9ebfb386912

File: .github/workflows/test.yml (hunk 2 above)
Comment (pychik) [reply .github/workflows/test.yml]: fixed in 6cec3ca1417d268a544349d1f4a3c9ebfb386912

**Review by vania-pooh — APPROVED** [review]

**Pull request merged**